src/productcatalogservice/products.json

```diff
@@ -107,6 +107,18 @@
         "nanos": 950000000
       },
       "categories": ["accessories", "telescopes"]
+    },
+    {
+      "id": "HQTGWGPNH4",
+      "name": "The Comet Book",
+      "description": "A 16th-century treatise on comets, created anonymously in Flanders (now northern France) and now held at the Universitätsbibliothek Kassel. Commonly known as The Comet Book (or Kometenbuch in German), its full title translates as “Comets and their General and Particular Meanings, According to Ptolomeé, Albumasar, Haly, Aliquind and other Astrologers”. The image is from https://publicdomainreview.org/collection/the-comet-book, made available by the Universitätsbibliothek Kassel under a CC-BY SA 4.0 license (https://creativecommons.org/licenses/by-sa/4.0/)",
```

Comment:
Do we need to keep this here?
```
The image is from https://publicdomainreview.org/collection/the-comet-book, made available by the Universitätsbibliothek Kassel under a CC-BY SA 4.0 license (https://creativecommons.org/licenses/by-sa/4.0/)
```

Reply:
@jlawrienyt just a comment here, if you say that we need to keep it in order to use the image, feel free to just resolve the conversation.
After that we should be good to merge.

Reply:
I added it based on the attribution rules listed in the CC license, so I think we need to keep it.
<img width="669" alt="image" src="https://user-images.githubusercontent.com/1426910/211578407-6b95ae2e-d74e-4a4b-a657-5ee65931b9ea.png">

Reply:
Perfect!